### PR TITLE
DIT-873 New suppress case journey

### PR DESCRIPTION
**Pull request by retroredge**

### CODE REVIEW

File: app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/contactDetails.scala.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.tariffclassificationfrontend.config.AppConfig
+@import uk.gov.hmrc.tariffclassificationfrontend.models.{Case, Contact}
+@import uk.gov.hmrc.tariffclassificationfrontend.models.request.AuthenticatedRequest
+
+@(c: Case)(implicit  messages: Messages)
```
Comment (ed0906) [inline app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/contactDetails.scala.html]: Can we make sure the relevant View tests are moved too if you haven't already :)

Comment (retroredge) [reply app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/contactDetails.scala.html]: Will do

Comment (retroredge) [reply app/uk/gov/hmrc/tariffclassificationfrontend/views/partials/contactDetails.scala.html]: I also found a 3rd place were we had a 'contact details' section in line in the template so I've pulled that out too

File: app/uk/gov/hmrc/tariffclassificationfrontend/views/confirm_supressed_case.scala.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+@main(bodyClasses = None) {
+    @case_heading(c)
+
+    <div class="grid-row">
+        <div class="column-full">
+            <h3 class="heading-large mt-0">This case has been suppressed</h3>
+
+            <div class="heading-medium mt-45">Next Steps</div>
+
+            <p>Go back to <a href="@uk.gov.hmrc.tariffclassificationfrontend.controllers.routes.QueuesController.queue("gateway")">@messages("cases.menu.gateway-cases")</a></p>
```
Comment (gokyo) [inline app/uk/gov/hmrc/tariffclassificationfrontend/views/confirm_supressed_case.scala.html]: `uk.gov.hmrc.tariffclassificationfrontend.controllers.routes.QueuesController` can be imported

Comment (retroredge) [reply app/uk/gov/hmrc/tariffclassificationfrontend/views/confirm_supressed_case.scala.html]: Done

**Pull request merged**